src/SearchExpression.cc

```diff
@@ -93,7 +93,8 @@ class SearchExpression::NodeAndIteratorImpl : public PMGD::NodeIteratorImplIntf
                                PMGD::StringID edgetag, bool unique,
                                const SearchExpression &neighbor_expr)
         : _expr(neighbor_expr),
-          mNodeIt(get_neighbors(node, dir, edgetag, unique)),
+          mNodeIt(get_neighbors(node, dir, edgetag,
```

Comment:
Our previous branches, we have split the PMGD QH level changes from the JSON level changes with unit tests at both levels. I suggest keeping to that. Helps identify breaking points in future commits.

Reply:
will skip testing at this level, given that I am already adding tests at VDMS and PMGD levels.

Reply:
Skipping tests is fine but please separate out the commits. It makes it easier to debug when something doesn't work right.